src/main/kotlin/no/nav/syfo/brev/arbeidstaker/domain/ArbeidstakerBrevSvarDTO.kt

```diff
@@ -0,0 +1,9 @@
+package no.nav.syfo.brev.arbeidstaker.domain
+
+import java.time.LocalDateTime
+
+data class ArbeidstakerBrevSvarDTO(
```

Comment:
Svar og Respons-dtoer er vel identiske for arbeidstaker og narmesteleder så kunne evt brukt en felles type og lagt den under no.nav.syfo.brev.domain f.eks.

Reply:
Ja, hadde vært mulig å slå sammen en del her, men jeg synes det er like greit å ha det adskilt.